### PR TITLE
[hi] L10n of Feedback section

### PR DESCRIPTION
**Pull request by Shivansh-yadav13**
- Related issue: #470

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Preview: https://deploy-preview-665--cncfglossary.netlify.app/hi/devops/

To take effect, please add `layouts/partials/feedback.html` file to this PR, as the same as [this](https://github.com/cncf/glossary/pull/483/files#diff-d979953f498f7a3c60d46cbd6649f6df538e7b6a0bb0b2d8663f962ae88df6d6).

Sorry for the previous insufficient guide 😅

[`layouts/partials/feedback.html` file to add]
```html
<style>
  .feedback--answer {
    display: inline-block;
  }
  .feedback--answer-no {
    margin-left: 1em;
  }
  .feedback--response {
    display: none;
    margin-top: 1em;
  }
  .feedback--response__visible {
    display: block;
  }
</style>
<div class="d-print-none">
<h2 class="feedback--title">{{ T "feedback_title" }}</h2>
<p class="feedback--question">{{ T "feedback_question" }}</p>
<button class="btn btn-primary mb-4 feedback--answer feedback--answer-yes">{{ T "feedback_answer_yes" }}</button>
<button class="btn btn-primary mb-4 feedback--answer feedback--answer-no">{{ T "feedback_answer_no" }}</button>
<p class="feedback--response feedback--response-yes">
  {{ .yes | safeHTML }}
</p>
<p class="feedback--response feedback--response-no">
  {{ .no | safeHTML }}
</p>
</div>
<script>
  const yesButton = document.querySelector('.feedback--answer-yes');
  const noButton = document.querySelector('.feedback--answer-no');
  const yesResponse = document.querySelector('.feedback--response-yes');
  const noResponse = document.querySelector('.feedback--response-no');
  const disableButtons = () => {
    yesButton.disabled = true;
    noButton.disabled = true;
  };
  const sendFeedback = (value) => {
    if (typeof ga !== 'function') return;
    const args = {
      command: 'send',
      hitType: 'event',
      category: 'Helpful',
      action: 'click',
      label: window.location.pathname,
      value: value
    };
    ga(args.command, args.hitType, args.category, args.action, args.label, args.value);
  };
  yesButton.addEventListener('click', () => {
    yesResponse.classList.add('feedback--response__visible');
    disableButtons();
    sendFeedback(1);
  });
  noButton.addEventListener('click', () => {
    noResponse.classList.add('feedback--response__visible');
    disableButtons();
    sendFeedback(0);
  });
</script>
```

**Comment by Shivansh-yadav13** [comment]
@jihoon-seo I was adding it previously but I wasn't sure to add it after referencing the #519, it might already exist there mb 😅

File: layouts/partials/feeback.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<style>
```
Comment (jihoon-seo) [inline layouts/partials/feeback.html:1]: Please rename this file to `feedback.html` . 😊

**Review by jihoon-seo — CHANGES_REQUESTED** [review]
@Garima-Negi @sayantani11 @anubha-v-ardhan @jayesh-srivastava Please review this PR, and perform merge if looks good. 😊

- Recommended merge method: 'Squash and merge'

File: layouts/partials/feeback.html (hunk 1 above)
Comment (Shivansh-yadav13) [reply layouts/partials/feeback.html]: 😅

**Review by jihoon-seo — APPROVED** [review]

**Review by Garima-Negi — APPROVED** [review]

**Comment by Garima-Negi** [comment]
LGTM!

**Pull request merged**